README.md

```diff
@@ -31,21 +31,30 @@ Have a bug or feature request? [File a bug on Launchpad][fileabug].
 Want to get involved in Mixxx development? Assign yourself a bug from the [easy
 bug list][easybugs] and get started!
 
-## Compiling
+## Compiling on Linux
+Please see our helpful guide on the [wiki] for more information: [Compiling on Linux]
 
-* macOS [![Build Status](https://travis-ci.org/mixxxdj/mixxx.svg)](https://travis-ci.org/mixxxdj/mixxx)
-* Ubuntu / Windows [![Build status](https://ci.appveyor.com/api/projects/status/j460rficblcaopwx?svg=true)](https://ci.appveyor.com/project/mixxxdj/mixxx)
-* Jenkins [![Build status](https://img.shields.io/jenkins/s/https/builds.mixxx.org/job/master-release.svg)](https://builds.mixxx.org/job/master-release)
+## Compiling on MacOS
+Please see our helpful guide on the [wiki] for more information: [Compiling on MacOS]
 
-First, you must install all of Mixxx's dependencies. To compile Mixxx using
-[SCons], run:
+## Compiling on Windows
+### Build Requirements
+- Windows 7 or later
+- MS Visual Studio 2019 (Community Edition is free of charge)
+- At least 10G free diskspace
 
-    $ scons
+### Setup your build environment
+1. Download these sources (using git checkout as described in [Using Git])
+2. Run the batch file `tools\windows_buildenv.bat`
+   - This file downloads the prebuild Mixxx environment, defined in `cmake\windows_build_environment_name` from https://downloads.mixxx.org/builds/buildserver/2.3.x-windows/
+   - Generates the `CMakeSettings.json` with the matching build configurations for Visual Studio
```

Comment:
Can you write instructions for building on the command line without coupling it to any particular IDE?

Reply:
I personally can't, because I don't know, how it works with command line. Maybe @Holzhaus can describe the command line steps needed for CI.
But anyway, this is out of scope of this PR and should be done in an own PR, or in the Wiki. Scope of this PR is to provide the VisualStudio IDE setup.

Reply:
I don't have windows so I can't test this. In theory you should be able to open a VS developer console, `CD` to the mixxx repository, then run:

    C:\mixxx\> tools\windows_buildenv.bat
    C:\mixxx\> MD build
    C:\mixxx\> CD build
    C:\mixxx\> cmake -G Ninja ..
    C:\mixxx\> cmake --build .